examples/spec/integration/failing_activities_workflow_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'workflows/failing_activities_workflow'
+
+describe FailingActivitiesWorkflow do
+  subject { described_class }
+  context 'async execution completes' do
+    let(:count) { 100 }
```

Comment:
How do you feel about making this number much lower and calling this in a controlled manned — for example stub `rand` and return expected results. Or maybe even design an example that does not depend on randomness?

Reply:
I'll make this deterministic and reduce the count. The randomness isn't really necessary and does make it harder to check the desired behavior is happening. I had it at 10 at one point, but didn't move it back down.